incremental-search/search-suggestions/src/main/application/deployment.xml

```diff
@@ -0,0 +1,20 @@
+<deployment version="1.0">
```

Comment:
This file can be removed

Reply:
will pull the new changes and remove the file